midi_app_controller/gui/utils.py

```diff
@@ -104,3 +137,25 @@ def get_selected_action_id(self) -> Optional[str]:
         """Returns the id of currently selected action."""
         if (action := self.currentData()) is not None:
             return action.id
+
+
+def is_subpath(path: Path, subpath: Path) -> bool:
+    """Checks if one path represents a file/directory inside the other directory."""
+    path_str = str(subpath.resolve().absolute())
+    subpath_str = str(path.resolve().absolute())
+    return path_str.startswith(subpath_str)
+
+
+def reveal_in_explorer(file: Path):
+    """Show where the file is in the system's file explorer.
+
+    Currently only Windows, Linux (majority of distributions), and macOS are supported.
+    """
+    if platform.system() == "Windows":
+        subprocess.Popen(["explorer", f'/select,"{file}"'])
+    elif platform.system() == "Linux":
+        subprocess.Popen(["xdg-open", f'"{file.parent}"'])
```

Comment:
It still doesn't work for me. It tries to open `"directory"` instead of `directory`. `Popen` already passes the string as a single argument, there is no need to use the `""`.